### PR TITLE
Readme updated

### PR DESCRIPTION
**Pull request by jwdebelius**
Per the discussion at the AG meeting today, updates the readme to explain about the new data location and why they're better.

### CODE REVIEW

**Comment by JWDebelius** [comment]
cc @EmbrietteH @wasade

**Comment by antgonza** [comment]
What about adding a big red banner about the old data here: https://github.com/JWDebelius/American-Gut/blob/readme_updated/data/README.md ?

**Comment by EmbrietteH** [comment]
I like @antgonza idea. The more obvious we make it, the better.

File: README.md [hunk 1]
```diff
@@ -1,4 +1,13 @@
 American-Gut
 ============
 
-American Gut open-access data and IPython notebooks
+American Gut open-access code and IPython notebooks
+
+A note about Data
+---
+
+American Gut sequences and metadata are deposited in [The European Bioinformatics Institute](http://www.ebi.ac.uk/ena) as ERP012803.
```
Comment (josenavas) [inline README.md]: `...are deposited in EBI as...` -> `...are deposited in EBI under the accession number...`
And make ERP012803 a link to the data: -> http://www.ebi.ac.uk/ena/data/view/ERP012803

File: README.md [hunk 2]
```diff
@@ -1,4 +1,13 @@
...
+
+The latest OTU tables and precalculated diversity comparisons generated by the primary processing notebook set can be found at [ftp://ftp.microbio.me/AmericanGut/](ftp://ftp.microbio.me/AmericanGut/).
```
Comment (josenavas) [inline README.md]: Change the link to ftp://ftp.microbio.me/AmericanGut/latest so it always point to the latest data.

File: README.md [hunk 3]
```diff
@@ -1,4 +1,13 @@
...
+
+Data currently hosted in this repository reflects the state of the project in May 2015 and before. This includes an earlier version of the American GUt survey and dietary questionaire. Data in GitHub has been scrubbed for PHI.
```
Comment (josenavas) [inline README.md]: `GUt` -> `Gut`

**Comment by josenavas** [comment]
Some comments on the text and I agree with @antgonza

File: README.md (hunk 3 above)
Comment (wasade) [inline README.md]: questionaire -> questionnaire

Comment (wasade) [inline README.md]: "Data currently hosted" -> "Data hosted"

Comment (wasade) [inline README.md]: Worth adding a mention of the bloom sequences?

Comment (EmbrietteH) [inline README.md]: Yes, that's a great idea.

File: README.md (hunk 2 above)
Comment (JWDebelius) [inline README.md]: This was deliberate; As the project grows, I think its better to let people select the dataset they wish to use. I think its easier when presented with all the data, over being forced into the latest set.

Comment (JWDebelius) [inline README.md]: But, I see your point given the fact that it says "Latest".

**Comment by JWDebelius** [comment]
@EmbrietteH: pinging to see if you're happy with this?

**Comment by EmbrietteH** [comment]
Can you address @antgonza suggestion which I and @josenavas agreed with? Doesn't have to be a red banner, but should be more obvious-bold text, red bold text, etc.-something to make it jump out so it's obvious. Otherwise it's good.

**Comment by JWDebelius** [comment]
It's as obvious as I can make it without violating good design conventions.

**Comment by EmbrietteH** [comment]
I'm not sure what you mean? I don't think bolding text to make it more eye catching violates any design conventions-at least none that I know of.

**Comment by josenavas** [comment]
Not an issue here - bolding will be good enough.

**Comment by jwdebelius** [comment]
ping @EmbrietteH

**Pull request merged**